### PR TITLE
Photo work

### PR DESCRIPTION
**Pull request by susannahmarcus**
[

](url)
Oct-01-2020 17-50-36

### CODE REVIEW

File: Database/seed.js [hunk 1]
```diff
@@ -23,6 +23,8 @@ const seed = () => {
     let hotelName = 'hotel' + i;
     let randNum = Math.floor(Math.random() * 58);
     let bool = (Math.random() > 0.5) ? true : false;
```
Comment (kcmcminn) [inline Database/seed.js:25]: Nothing important, but if you want to, Faker has a random boolean generator too! I used like 50 of them in mine :-D

File: Database/seed.js [hunk 2]
```diff
@@ -35,7 +37,14 @@ const seed = () => {
       image.user = faker.name.firstName() + faker.name.lastName();
       image.hotel = hotelName;
       image.tag = tags[tagsNum];
-      image.special = bool;
+      specialObj.is = bool;
+
+      if (bool) {
+        specialObj.specialItem = gamble > .5 ? `https://cdn.wallpapersafari.com/99/98/2IjALn.jpg` : `https://www.tripadvisor.com/Hotel_Review-g150807-d152680-Reviews-Fiesta_Americana_Condesa_Cancun_All_Inclusive-Cancun_Yucatan_Peninsula.html#/media/152680/316101332:v/?albumid=14&type=0&category=14`;
```
Comment (kcmcminn) [inline Database/seed.js]: This reminds me that I need to use ternary operators more often!

**Review by kcmcminn — APPROVED** [review]
Looks good!

File: Database/seed.js (hunk 1 above)
Comment (susannahmarcus) [reply Database/seed.js:25]: Thanks for the heads up! I didn't know realize that!

File: Database/seed.js (hunk 2 above)
Comment (susannahmarcus) [reply Database/seed.js]: Be careful with that. Annah has said in the past that people shouldn't use them very often, but I like them so I do... They work well with some react issues, so it's always good to have in one's back pocket.

**Pull request merged**